config.go

```diff
@@ -27,9 +27,9 @@ var (
 
 type Volumes []Volume
 type Volume struct {
-	Secret    *string `yaml:"secret,omitempty" jsonschema:"oneof_required=secret"`
-	ConfigMap *string `yaml:"configMap,omitempty" jsonschema:"oneof_required=configmap"`
-	Path      *string `yaml:"path"`
+	Secret    *string `json:"secret,omitempty" jsonschema:"oneof_required=secret"`
```

Comment:
It would be better if this change from `yaml` to `json`all across this PR would be done in a seperate PR, with some description and explanation. If I am not mistaken it is not relevant to probes.

Reply:
That would be great!